src/ApiBundle/Controller/AuthController.php

```diff
@@ -248,23 +248,7 @@ public function postRegisterAction()
         $errors = $validator->validate($user);
 
         if (count($errors) > 0) {
-            $errorArray = [];
-            foreach ($errors as $error) {
-                $constraint = $error->getConstraint();
-                $errorItem = array(
-                                    "error_description" => $error->getPropertyPath().': '.$error->getMessage().' '.$error->getInvalidValue(),
-                                    "show_message" => $this->get('translator')->trans($constraint->payload['api_error'], array(), 'messages', $request->getLocale())
-                                  );
-                array_push($errorArray, $errorItem);
-                $this->logMessage(400, $errorItem['error_description'] );
-            }
-            return new JsonResponse(array(
-                          "code" => 400,
-                          "error" =>  "Bad Request",
-                          "error_description" => $errorArray[0]['error_description'],
-                          "show_message" => $errorArray[0]['show_message'],
-                          'errors' => $errorArray
-                      ));
+           return $this->reportValidationErrors($request, $errors);
```

Comment:
`+           return $this->reportValidationErrors($errors, $request->); `
can we make it,  
`+           return $this->reportValidationErrors($errors, $request->getLocale()); `